### PR TITLE
Gpg sign if not jarsigned

### PR DESCRIPTION
**Pull request by akurtakov**

### CODE REVIEW

**Comment by akurtakov** [comment]
@merks @leods This would fix the missing signatures for org.jsoup . I think  all signatures should be fine now but please let me know if smth is still not correct. https://download.eclipse.org/mylyn/snapshots/nightly/docs/ should have the latest in half an hour.

File: docs/org.eclipse.mylyn.docs-site/pom.xml [hunk 1]
```diff
@@ -11,4 +11,30 @@
...
+						<artifactId>tycho-gpg-plugin</artifactId>
+						<version>${tycho-version}</version>
+						<executions>
+							<execution>
+								<id>pgpsigner</id>
+								<goals>
+									<goal>sign-p2-artifacts</goal>
+								</goals>
+								<configuration>
+									<keyname>0b8ae29554d3b0c1</keyname>
```
Comment (ruspl-afed) [inline docs/org.eclipse.mylyn.docs-site/pom.xml:30]: @akurtakov should I reuse this literally for all the rest of mylyn repositories?

Comment (akurtakov) [reply docs/org.eclipse.mylyn.docs-site/pom.xml:30]: I think it should work yes but note that the Jenkinsfile part is equally important. Also this is needed only if you have non jarsigned content - in docs case this is org.jsoup

**Comment by merks** [comment]
@akurtakov Yes, this looks good now:

https://download.eclipse.org/oomph/archive/reports-extra/mylyn-snapshots/download.eclipse.org/mylyn/snapshots/nightly/docs/index.html


image

**Comment by leods** [comment]
Sure. I'd hoped that we'd have the old release jobs auto-migrated from ci/docs to ci/mylyn, but I'm going to work on recreating the release job and with Brandon's help we'll try to have something ready soon.

**Comment by brandonjacob** [comment]
@akurtakov, thanks for fixing up the PGP signing.  That's wonderful!

As for the mylyn release job, I'll starting taking a look.  I will explore the effort for creating a jenkins pipeline job (and associating jenkinsfile) that could be used to replace the old, manually-created release jobs.  Obviously getting the build logic into source control would be a significant improvement over the previous manually generated jobs.  I think I can craft a derivative that is based on the existing jenkinsfile you've created for building and deploying snapshots.

**Comment by leods** [comment]
@akurtakov I've created a new pipeline job at https://ci.eclipse.org/mylyn/view/Docs/job/github-mylyn-docs-release/ and I'd like for @brandonjacob to be able to help out with setting this up, but as far as I can tell there's now way for me to edit user permissions to allow him to edit this job. Are permissions for the pipeline jobs managed differently than the freestyle jenkins jobs? Do you know if it's possible for a contributor/non-committer to get edit permissions on some of these pipeline jobs?

**Comment by akurtakov** [comment]
Isn't pipeline job managed by jenkinsfile? In this case pr is due. No idea otherwise.

**Comment by leods** [comment]
Ignore, I found what I was looking for.

**Comment by brandonjacob** [comment]
I can view/edit the configuration for the new [candidate pipeline job](https://ci.eclipse.org/mylyn/view/Docs/job/github-mylyn-docs-release/) that @leods created.  I'll start exploring a jenkinsfile derivative for running the mylyn docs release build steps.

**Comment by brandonjacob** [comment]
@leods and/or @akurtakov :  I think I'm close with a jenkins pipeline job that can produce mylyn docs releases.  I've been incrementally working on a new jenkinsfile and doing some testing with the associating [job](https://ci.eclipse.org/mylyn/view/Docs/job/github-mylyn-docs-release/).  

While starting to test staging/deployment, I ran into a [problem](https://ci.eclipse.org/mylyn/view/Docs/job/github-mylyn-docs-release/20/console) with the nexus-staging plugin configuration.  I think this is happening because the default maven settings.xml file doesn't have the ossrh configuration as indicated in this [documentation](https://wiki.eclipse.org/Tycho:How_to_deploy_to_a_Maven_repository).  Would we need to submit a ticket to ask the Eclipse IT team to configure this part?

**Comment by akurtakov** [comment]
I have zero experience with ossrh but if you need change in default settings.xml opening ticket is best starting point.

**Comment by brandonjacob** [comment]
Fair enough. I've posted a comment under a currently [open ticket here](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2268).  If necessary, I'll open a new ticket to get the OSSRH configuration applied to the default maven settings.xml file.

**Pull request merged**